### ISSUE TITLE
Cannot install on latest version of inertia

### ISSUE THREAD

**Issue by AUAboi** [laravel-arcanist/inertia-response-renderer#7]
On installation I get:
```
- Root composer.json requires laravel-arcanist/inertia-response-renderer ^0.5.0 -> satisfiable by laravel-arcanist/inertia-response-renderer[0.5.0].
    - laravel-arcanist/inertia-response-renderer 0.5.0 requires inertiajs/inertia-laravel ^0.4.1 -> found inertiajs/inertia-laravel[v0.4.1, 
..., v0.4.5] but it conflicts with your root composer.json require (^0.6.2).
```

**Comment by alslimany** [comment]
Same issue, the arcanist response renderer must support inertiajs/inertia-larave:^0.5.2

**Comment by ksassnowski** [comment]
Fixed in 0.6.0
